tensorflow/c/kernels.h

```diff
@@ -65,6 +66,11 @@ typedef struct TF_KernelBuilder TF_KernelBuilder;
 typedef struct TF_OpKernelConstruction TF_OpKernelConstruction;
 typedef struct TF_OpKernelContext TF_OpKernelContext;
 
+// TF_InitKernel to do op/kernel registration.
+// Plugin should either implement TF_InitKernel to register kernels or use
+// static registration. This function should register all kernels in a plugin.
```

Comment:
Let's remove mention of "static registration" per Sanjoy's comment on this PR: https://github.com/tensorflow/tensorflow/pull/43610/files

Reply:
done